README.md

```diff
@@ -39,17 +32,22 @@ There is a new Max. resolution select field in the inputstream.adaptive settings
 Auto will select the best resolution matching to your videoplayer display rect without any limits.
 If your display resolution is 720p, you will not be able to watch 1080p videos if there are video representations available closer to 720p.
 
-
-##### TODO's:
-- Adaptive bitrate switching is prepared but currently not yet activated
+#### TODO's:
+- Adaptive bitrate switching has been enabled on Kodi 20 but need more improvements
 - Automatic / fixed video stream selection depending on max. visible display rect (some work has to be done at the inputstream interface).
-- There will be many dash mpd, smoothstream or hls manifest types currently not supported - must be extended.
+- There will be many MPEG-DASH, HSL or Smooth Stream manifest types currently not supported that must be extended.
 
-##### Notes:
+### Notes:
 - This addon uses threads to download segments. The memory consumption is the sum of single segment from each stream currently playing. Refering to known streams it is < 10MB for 720p videos.
 
-##### Credits:
-[@peak3d](https://github.com/peak3d) Original author / creator. Superstar!
-[@fernetmenta](https://github.com/fernetmenta) Best support regarding streams / codecs and kodi internals.
-[@notspiff](https://github.com/notspiff) Ideas / tips regarding kodi file system.
-[bento4 library](https://www.bento4.com/) Great library for mp4 streams. Well written and extensible!
+## Acknowledgements
+InputStream Adaptive exists thanks to the help of many
+
+* Special thanks to the author / creator [@peak3d](https://github.com/peak3d)
```

Comment:
Missing CastagnaIT and glenn guy here :)

Reply:
i do not know if we should add each new developer added over time
we should therefore compile a longer list and this is not something I would like to maintain...

peak3d is mentioned because this initially this was developed by him only,
IMO i would only keep the first one and the last two points (peak3d/contributors/bento)
all others kodi devs they should be implicit, or not?